Add instructions for connecting on the internal network (#362)
space-wizards/docs: src/en/general-development/setup/server-hosting-tutorial.md

{% embed youtube id="IDBqrAGZ3cA" loading="lazy" %}

## Level 1: Invite Your Friends/Family

If you are looking to let people from your *internal* network (connected to the same router as you).

#### Getting your internal IP address (Windows)
1. Open Windows Settings
2. Click on "Network & Internet"

> 3. Look for "IPV4 address". Thats your computers local IP. And what your family needs to enter in the direct connect menu to join.

Reviewer: ```suggestion
3. Look for "IPV4 address". That's your computers local IP. And what people on your local network needs to enter into the direct connect menu to join.
```
